1회차/이유주/1_모음이보이지않는사람.py

```diff
@@ -1,3 +1,14 @@
 import sys
 
 sys.stdin = open("_모음이보이지않는사람.txt")
+
+
+vowel = ['a', 'e', 'o', 'u', 'i']
+T = int(input())
+for i in range(1, T+1):
+    word = input()
+    for x in word:
+        if x in vowel:
+            word = word.replace(x, '')
```

Comment:
`replace` 함수를 활용한 좋은 코드네요! 👍 
아래와 같은 방법으로도 해결이 가능합니다
```suggestion
    for x in vowel:
        word = word.replace(x, '')
```